extensions/Cake.Addin3.yml

```diff
@@ -0,0 +1,16 @@
+Type: Addin
+Name: Cake.Addin3
+NuGet: Cake.Addin3
```

Comment:
This one looks like a candidate for the exclusion list. The description is "TestPackage" and the repo doesn't seem to be anywhere on GitHub

Reply:
Adding this addin to exclusion list: https://github.com/cake-contrib/Cake.AddinDiscoverer/issues/194